sinopia_indexer/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "sinopia_indexer",
+  "version": "0.1.0",
+  "description": "Sinopia Indexer listens for STOMP messages, parses URIs out of messages, retrieves resources from an LDP server, and indexes them into ElasticSearch ",
+  "license": "Apache-2.0",
```

Comment:
You should add a `contributors` hash with your name!

Reply:
Will do!